modules/unsupported/geojson-core/src/test/java/org/geotools/data/geojson/GeoJSONWriteTest.java

```diff
@@ -297,7 +299,12 @@ public void testDateFormat() throws Exception {
         String json = out.toString(StandardCharsets.UTF_8);
         JsonNode root = new ObjectMapper().readTree(json);
         String date = root.get("properties").get("date").textValue();
-        assertEquals(DateFormatUtils.ISO_8601_EXTENDED_DATE_FORMAT.format(testDate), date);
+
+        FastDateFormat utm =
+                FastDateFormat.getInstance(
+                        DateFormatUtils.ISO_8601_EXTENDED_DATE_FORMAT.getPattern(),
+                        GeoJSONWriter.DEFAULT_TIME_ZONE);
```

Comment:
writer.setDatePattern method above reuses the timezone of the existing writer formatter, which by default is DEFAULT_TIME_ZONE. 

Using GMT is correct as I understand it for JSON, so I am correcting the test case with this in mind.